source/thunkify.js

```diff
@@ -0,0 +1,29 @@
+import _curry1 from './internal/_curry1';
+import _arity from './internal/_arity';
+
+/**
+ * Creates a thunk out of a function. A thunk delays a calculation until
+ * its result is needed, providing lazy evaluation of arguments.
+ *
+ * @func
+ * @memberOf R
+ * @category Function
+ * @sig (a -> b) -> (a -> b) // FIXME: write appropiate signature
+ * @param {Function} fn A function to wrap in a thunk
+ * @return {Function} Expects arguments for `fn` and returns a new function
+ *  that, when called, applies those arguments to `fn`.
+ * @see R.partial, R.partialRight
+ * @example
+ *
+ *      R.thunkify(R.identity)(42)(); //=> 42
```

Comment:
I'd like to see at least one additional example here that uses more than one parameter.

Reply:
👍. Anything in particular in mind?

Reply:
Even a simple `thunkify((a, b) => a + b)(25, 17)() ==> 42` would help.